src/python/core.py

```diff
@@ -9,6 +9,8 @@
 
 from .om import Warning
 
+binary_default = "../output/bin/iec_checker"
```

Comment:
Hmm, maybe we should replace it with "os.path. join" so that it works on Windows.

```suggestion
binary_default = os.path.join("..", "output", "bin", "iec_checker")
```